### ISSUE TITLE
`code-insiders-1.94.0-1725516996.el8.rpm` copies random text, not selection.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [microsoft/vscode#227822]
<table><tr>
	<th>Does this issue occur when all extensions are disabled?</th>
	<td>Yes</td>
</tr></table>

<table><tr>
	<th>VS Code Version</th>
	<td><code>1.94.0-1725516996.el8</code></td>
</tr><tr>
	<th>OS Version</th>
	<td><a href='https://download.fedoraproject.org/pub/fedora/linux/releases/40/Spins/x86_64/iso/Fedora-KDE-Live-x86_64-40-1.14.iso'><code>cpe:/o:fedoraproject:fedora:40</code></a></td>
</tr></table>

This *does not* occur on `code-1.93.0-1725459174.el8.rpm`.

#### **Steps to Reproduce**

Invoke the undermentioned code:

```sh
#!/usr/bin/env -S sh
sudo dnf install 'code-insiders-1.94.0-1725516996.el8' && \
code-insiders --disable-extensions
```

...as the undermentioned video demonstrates:

https://github.com/user-attachments/assets/26ecb11e-1001-42a0-8519-a07b9ea05a9e

Apologies for the poor video - I was so focussed trying to work out what was going on that I neglected to disable the webcam. Luckily, it doesn't obscure much of importance.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/microsoft/vscode/issues/227822#issue-2510906813

Per https://github.com/microsoft/vscode/issues/227811#issuecomment-2334454888, is this a duplicate of https://github.com/microsoft/vscode/issues/227721#issue-2508539886 too?

**Comment by roblourens** [comment]
Yes, https://github.com/microsoft/vscode/issues/227721

**Comment by RokeJulianLockhart** [comment]
Fixed in `code-insiders-1.94.0-1725631051.el8.rpm`.
